pkg/cmd/server/origin/auth.go

```diff
@@ -355,8 +357,11 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 				}
 				passwordSuccessHandler := handlers.AuthenticationSuccessHandlers{c.SessionAuth, redirectSuccessHandler{}}
 
+				loginPath := path.Join(OpenShiftLoginPrefix, identityProvider.Name)
```

Comment:
Do we want to give the first one the `OpenShiftLoginPrefix` or only if it is the only provider? I'm inclined to say that if there are multiple providers, we should not arbitrarily give one an exceptional status, but if you have a strong argument, I'll hear it.